TeamCode/src/main/java/org/firstinspires/ftc/teamcode/ftc16072/BehaviorTrees/Conditions/IsLastThreeSeconds.java

```diff
@@ -0,0 +1,14 @@
+package org.firstinspires.ftc.teamcode.ftc16072.BehaviorTrees.Conditions;
+
+import org.firstinspires.ftc.teamcode.ftc16072.BehaviorTrees.Node;
+import org.firstinspires.ftc.teamcode.ftc16072.OpModes.QQOpMode;
+
+public class IsLastThreeSeconds extends Node {
+    @Override
+    public State tick(QQOpMode opmode) {
+        if(opmode.time > 117){
```

Comment:
Replace 117 with NAMED_CONSTANT that uses math.  For example LAST_THREE_SECONDS = TELEOP_TIME - 3;